README.md

```diff
@@ -109,10 +109,6 @@ your project (for example, at `bin/venv-update`).
 Then, apply a change like this to your `tox.ini` file:
 
 ```patch
-  [tox]
-  envlist = py27,py34
-+ skipsdist = true
-
```

Comment:
Probably belongs in the doc.

Also, sad. The sdist testing is really the only part of tox that I think is a good idea that I wouldn't have done myself.